modules/openapi-generator/src/main/resources/python/model.mustache

```diff
@@ -221,6 +221,40 @@ class {{classname}}(object):
 
         return result
 
+    def to_json_dict(self):
+        """Returns the model properties as a dict"""
+        result = {}
+
+        def get_value(val):
+            if hasattr(val, "to_json_dict"):
+                return val.to_json_dict()
+            elif hasattr(val, "to_dict"):
+                return val.to_dict()
+            else:
+                return val
+
+        for attr, _ in six.iteritems(self.openapi_types):
+            value = getattr(self, attr)
+            key = self.attribute_map.get(attr, attr)
+            if isinstance(value, list):
+                result[key] = list(map(
+                    lambda x: get_value(x),
+                    value
+                ))
+            elif hasattr(value, "to_json_dict"):
+                result[key] = value.to_json_dict()
+            elif hasattr(value, "to_dict"):
+                result[key] = value.to_dict()
+            elif isinstance(value, dict):
+                result[key] = dict(map(
+                    lambda item: (item[0], get_value(item[1])),
+                    value.items()
+                ))
+            else:
+                result[key] = value
+
+        return result
+
```

Comment:
This code looks good but it is being added to every model that is generated?
How about centralizing a utility fn in model_utils.py and using it in each class?
That's what we do in python_experimental and it minimizes code duplication.
- modelutils.model_to_dict https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache#L1188
- using it in a model template https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/python/python-experimental/model_templates/methods_todict_tostr_eq_shared.mustache#L1
- example output model method https://github.com/OpenAPITools/openapi-generator/blob/master/samples/openapi3/client/petstore/python-experimental/petstore_api/model_utils.py#L369

Reply:
Hi @spacether ，have updated the PR.  I modified the to_dict function instead of adding a new function, and it is compatible to those sdk which is not generated using the new code. Please help to review.